packages/camera/camera_android_camerax/android/src/main/java/io/flutter/plugins/camerax/LiveDataFlutterApiWrapper.java

```diff
@@ -0,0 +1,58 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.plugins.camerax;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.VisibleForTesting;
+import androidx.lifecycle.LiveData;
+import io.flutter.plugin.common.BinaryMessenger;
+import io.flutter.plugins.camerax.GeneratedCameraXLibrary.LiveDataFlutterApi;
+
+/**
+ * Flutter API implementation for {@link LiveData}.
+ *
+ * <p>This class may handle adding native instances that are attached to a Dart instance or passing
+ * arguments of callbacks methods to a Dart instance.
+ */
+public class LiveDataFlutterApiWrapper {
```

Comment:
Non blocking but if these are wrapper classes I wonder why we don't add the generation of these to the utility of pigeon. @bparrishMines Are these classes created from your generator of so what do you think about including it in pigeon?

Reply:
Adding the conversions of identifier -> Objects and Objects -> identifiers as a part of pigeon is one of my goals for this year. It's a little tricky because we would need to import the converted class for native code and dart. Or we could try using generics.